src/build/__tests__/build-test.js

```diff
@@ -149,4 +164,11 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using SCHEMA_CONFIG_KEY', () => {
```

Comment:
Please change `directives` to `schema` in the test description and I'll merge! 😄 

By the way, the reason for this change is because this change/test actually works for any schema configuration and not just directives.